SPECS/ocaml-inotify.spec

```diff
@@ -1,29 +1,21 @@
-%define opt %(test -x %{_bindir}/ocamlopt && echo 1 || echo 0)
-%define debug_package %{nil}
-
 Name:           ocaml-inotify
-Version:        2.0
+Version:        2.3
 Release:        1%{?dist}
 Summary:        Inotify bindings for OCaml.
 
 Group:          Development/Libraries
-License:        Apache Software License 2.0
+License:        GPL 2.1
```

Comment:
The LICENSE file in the repository seems to be LGPL, not GPL.  According to https://fedoraproject.org/wiki/Licensing:Main?rd=Licensing, the correct identifier is "LGPLv2 with exceptions".